app/views/protocols/show.html.haml

```diff
@@ -2,32 +2,32 @@
 %div{role: 'tabpanel'}
   %ul.nav.nav-tabs.nav-justified{role: 'tablist'}
     - if @services_present
-      %li.active.custom-tab.protocol-tab{role: 'presentation'}
+      %li.custom-tab.protocol-tab{role: 'presentation', class: (tab = cookies['active-protocol-tab'.to_sym]) ? ((tab == "study_schedule") ? "active" : "") : "active"}
```

Comment:
I would think about refactoring these inline conditions to a view helper to DRY up code